go/pkg/clickhouse/schema/002_key_verifications_per_minute_v2.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TABLE key_verifications_per_minute_v2 (
+  time DateTime,
+  workspace_id String,
+  key_space_id String,
+  identity_id String,
+  key_id String,
+  outcome LowCardinality (String),
+  tags Array(String),
+  count SimpleAggregateFunction(sum, Int64),
+  spent_credits SimpleAggregateFunction(sum, Int64),
+  latency_avg AggregateFunction (avg, Float64),
+  latency_p75 AggregateFunction (quantilesTDigest (0.75), Float64),
+  latency_p99 AggregateFunction (quantilesTDigest (0.99), Float64),
```

Comment:
I can't query any of the MV's 

[HY000] Failed to read value for column latency_avg
Only native integer types are supported but we got: Float64

same for ratelimits and key verifications